CONTRIBUTING.md

```diff
@@ -33,26 +35,32 @@ A single resource has the following structure, where all attributes are required
 You can now go ahead and make changes to this file, using one of the following option.
 
 ### Using the UI
-Navigate to the `contnet` directory and click on the `resources.json` file. Here you can click on the pencil icon to edit the file. 
 
-![grafik](https://user-images.githubusercontent.com/28757404/142865728-4d26e83a-8e60-4c33-809b-02198fa0506f.png)
+Navigate to the `content` directory and click on the `resources.json` file. Here you can click on the pencil icon to edit the file.
 
-You can now make your changes right inside the browser. Once you are done commit your changes by using the dedicated section below the displayed file content.
+![graphic](https://user-images.githubusercontent.com/28757404/142865728-4d26e83a-8e60-4c33-809b-02198fa0506f.png)
 
-![grafik](https://user-images.githubusercontent.com/28757404/142865981-a8be3047-26ce-45dc-82b6-1318887dadde.png)
+You can now make your changes right inside the browser. Once you are done commit your changes by using the dedicated section below the displayed file content.
 
+![graphic](https://user-images.githubusercontent.com/28757404/142865981-a8be3047-26ce-45dc-82b6-1318887dadde.png)
 
 ### Make changes locally
-Once you have forked the repository, go ahead and clone it locally. 
+
+Once you have forked the repository, go ahead and clone it locally.
 You need to install or update [node.js](https://nodejs.org/en/)
```

Comment:
suggestion(non-blocking): we should mention which min version of nodejs is required when we suggest updating nodejs.